plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryTypeMapping.java

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
This looks like a good starting point and covers how users will be using BigQuery connector (insert via BQ, read via Trino).